runtime/lua/vim/treesitter.lua

```diff
@@ -94,25 +94,43 @@ function M.get_parser(bufnr, lang, opts)
 
   if not valid_lang(lang) then
     if not parsers[bufnr] then
-      error(
-        string.format(
-          'There is no parser available for buffer %d and one could not be'
-            .. ' created because lang could not be determined. Either pass lang'
-            .. ' or set the buffer filetype',
```

Comment:
Should this message be kept as a `vim.notify`? It would cause extra noise in e.g. `_comment.lua`